ci-operator/config/openshift/windows-machine-config-bootstrapper/openshift-windows-machine-config-bootstrapper-community-4.6__okd.yaml

```diff
@@ -0,0 +1,67 @@
+base_images:
+  ocp_builder_rhel-8-golang-1.15-openshift-4.6:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.15-openshift-4.6
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.15
+images:
+- dockerfile_path: Dockerfile.tools
+  inputs:
+    ocp_builder_rhel-8-golang-1.15-openshift-4.6:
+      as:
+      - registry.svc.ci.openshift.org/ocp/builder:rhel-8-golang-1.15-openshift-4.6
+  to: windows-machine-config-bootstrapper-tests
+promotion:
+  name: "4.6"
+  namespace: okd
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tag_specification:
+  name: "4.6"
+  namespace: origin
+test_binary_build_commands: make build GO_BUILD_FLAGS:='-race' --warn-undefined-variables
+tests:
+- as: build
+  commands: |
+    make build --warn-undefined-variables
```

Comment:
What the purpose of `--warn-undefined-variables` in this case?

Reply:
Applies to `make verify-all` as well

Reply:
I am not sure about a particular case. Keeping it consistent with other config [files](https://github.com/openshift/release/blob/master/ci-operator/config/openshift/windows-machine-config-bootstrapper/openshift-windows-machine-config-bootstrapper-master.yaml#L41)

Reply:
I think it's part of make, please look at https://www.oreilly.com/library/view/managing-projects-with/0596006101/ch12.html

Reply:
I don't think @sebsoto's question is about what `--warn-undefined-variables` does but **why** is it being added here. And @VaishnaviHire's answer is consistency. I vaguely remember that `--warn-undefined-variables` was not added by us but by the release team and @VaishnaviHire is just following that pattern.